lib/linter/apply-disable-directives.js

```diff
@@ -261,17 +261,21 @@ function applyDirectives(options) {
     const processed = processUnusedDisableDirectives(unusedDisableDirectivesToReport);
 
     const unusedDisableDirectives = processed
-        .map(({ description, fix, position }) => ({
-            ruleId: null,
-            message: description
-                ? `Unused eslint-disable directive (no problems were reported from ${description}).`
-                : "Unused eslint-disable directive (no problems were reported).",
-            line: position.line,
-            column: position.column,
-            severity: options.reportUnusedDisableDirectives === "warn" ? 1 : 2,
-            nodeType: null,
-            ...options.disableFixes ? {} : { fix }
-        }));
+        .map(({ description, fix, position }) => {
+            const { parentComment, type, line, column } = position;
```

Comment:
It looks confusing to get `parentComment` and `type` from a property named `position`. Since that object is actually the original directive object passed in by the linter, maybe we could rename the property to `unprocessedDirective` (as that's how it is called in the rest of the code)?

Reply:
Done.